src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,44 @@
 package core.basesyntax;
 
-class Pair {
+public class Pair<T1, T2> {
```

Comment:
https://stackoverflow.com/questions/2900881/generic-type-parameter-naming-convention-for-java-with-multiple-

Reply:
I see, 
Updated